docs/source/user-guide/cli/datasources.md

```diff
@@ -95,8 +95,7 @@ additional configuration options.
 # `CREATE EXTERNAL TABLE`
 
 It is also possible to create a table backed by files or remote locations via
-`CREATE EXTERNAL TABLE` as shown below. Note that wildcards (e.g. `*`) are also
-supported
+`CREATE EXTERNAL TABLE` as shown below. Note that DataFusion does not support wildcards (e.g. `*`) in file paths; instead, specify the directory path directly to read all compatible files in that directory.
```

Comment:
Can you add an example? I'd say "given this file structure with these files (including a hive parittion column) configure an extrenal table this way and if you run `select * from t1` you'll get back something like this <insert table of resutls>